pom.xml

```diff
@@ -86,7 +86,7 @@
         <licenses.version>5.2.5-SNAPSHOT</licenses.version>
         <netty.version>4.1.63.Final</netty.version>
         <parquet.version>1.11.0</parquet.version>
-        <thrift.version>0.9.3</thrift.version>
+        <thrift.version>0.9.3-1</thrift.version>
```

Comment:
This leaves 4 CVE open still. 
We need 0.13.0 which I believe works will will leave only one open